index.html

```diff
@@ -46,6 +46,204 @@ <h1>I'm Naan.
+                                <div class="client">
+                                    <div class="client_text ">Canopy</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="role">
+                                    <div class="text underline">Back End Dev</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year_label">
+                                    <div class="text underline">2015</div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class ="primary_text_1">A daily selection of 
+                            privately personalized reads; no accounts or 
+                            sign-ups required.
+                        </div>
+                        <div class="tags">
+                            <div class="tag_1 tag_html">html</div>
+                            <div class="tag_2 tag_css">css</div>
+                            <div class="tag_3 tag_javascript">javascript</div>
+                        </div>
+                        <div class="actions">
+                            <div class="buttons">
+                                <div class="see_project">See project</div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div>
+                <div class="about_myself">
+                    <div class="top underline">
+                        <div class="about_header underline">About Myself</div>
+                        <div class="about_description ">
+                            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                        </div>
+                        <div class="social_media">
+                            <div class="supporting_text underline">
+                                <div class="connect ">LET’S CONNECT</div>
+                            </div>
+                        </div>
+                        <div class="icon_container border">
+                            <div><img src="images/github.jpg"></div>
+                            <div><img src="images/linkedin.jpg"></div>
+                            <div><img src="images/reddit.jpg"> </div>
+                            <div><img src="images/twitter.jpg"> </div>
+                            <div><img src="images/medium.jpg"> </div>
+                        </div>
+                        <div class="resume_button">
+                            <div class="resume_text underline">Get my resume</div>
```

Comment:
You have designed it correctly, but I have seen some issues in this section. Enlisting them below:
1. The space between the two sections is quite large. Kindly decrease it by comparing it with the Figma design.
2. You are using `font-size: 16px`. The size in Figma design is `font-size: 14px`.
3. You are using `font-size: 32px`. The size in Figma design is `font-size: 20px`.
4. The Upper down arrow is missing in your design while the two arrows of framework and skills are a bit left in your design. See the comparison below.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219898485-777dc44e-c2aa-48e9-b3b9-a3b75702e4ff.png)| ![image](https://user-images.githubusercontent.com/61361037/219897070-8ed57b75-41b6-4d0d-91f2-7a502709a54e.png)|